games-util/pyfa/pyfa-2.9.4.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+PYTHON_COMPAT=( python3_6 )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit desktop eutils gnome2-utils python-r1
+
+DESCRIPTION="Python Fitting Assistant - a ship fitting application for EVE Online"
+HOMEPAGE="https://github.com/pyfa-org/Pyfa"
+
+RESTRICT="mirror bindist"
+LICENSE="GPL-3+ all-rights-reserved"
+SLOT="0"
+if [[ ${PV} = 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/pyfa-org/Pyfa.git"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/pyfa-org/Pyfa/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+IUSE="+graph"
```

Comment:
I see upstream has enabled tests, have you tried implementing those in the ebuild? Ie, do they work?

Reply:
I tried to figure out how the the tests work, but it does not look like they run them in the CI either, just some packaging steps. I'm currently not sure how to run them and if they would even work.